app/graph/types/project_type.rb

```diff
@@ -1,39 +1,39 @@
 ProjectType = GraphqlCrudOperations.define_default_type do
   name 'Project'
-  description 'Project type'
+  description 'A collection of media items in a team.'
 
   interfaces [NodeIdentification.interface]
 
-  field :avatar, types.String
-  field :description, types.String
-  field :title, !types.String
-  field :dbid, types.Int
-  field :permissions, types.String
-  field :get_slack_channel, types.String
-  field :pusher_channel, types.String
-  field :medias_count, types.Int
-  field :search_id, types.String
-  field :url, types.String
-  field :search, CheckSearchType
-  field :auto_tasks, JsonStringType
-  field :team, TeamType
+  field :avatar, types.String, 'Picture' # TODO Rename to 'picture'
+  field :description, types.String, 'Description'
+  field :title, !types.String, 'Name' # TODO Rename to 'name'
+  field :get_slack_channel, types.String, 'Slack channel to notify about this project activity' # TODO Rename to 'slack_channel'
+  field :pusher_channel, types.String, 'Channel for push notifications'
+  field :medias_count, types.Int, 'Count of items in this collection'
+  field :search_id, types.String # TODO Document
+  field :url, types.String, 'Collection permalink'
+  field :search, CheckSearchType, 'Search interface'
+  field :auto_tasks, JsonStringType # TODO Why is this here?
```

Comment:
I think it shouldn't be, and would probably crash if called, because I can't see a method definition for it anywhere.